frontend/src/views/Collection/index.tsx

```diff
@@ -76,6 +78,10 @@ const Collection: FC<Props> = ({ id = "" }) => {
 
   const [mutate] = useCollectionUploadLinks(id, visibility);
 
+  const [isAlertOpen, openAlert] = useState(false);
+
+  const [deleteDataset] = useDeleteDataset(collection?.id ?? "");
```

Comment:
is there a reason we need `??` instead of just `||` here 😄 ?

Reply:
Just to be more specific that we're setting it to "" when collection is nullish.  Would you prefer:
```suggestion
  const [deleteDataset] = useDeleteDataset(collection?.id || "");
```

Reply:
Hmm yeah personally when I see `??`, it makes me wonder what other possible falsey values `collection?.id` could have that we **don't** want to use the fallback value `""`. Such as `0`, `false`, and `""`.

But in our case, I think the intention is to use the fallback value for all falsey values, thus `||`?